core-common/src/main/java/org/mapstruct/MapperConfig.java

```diff
@@ -88,5 +88,16 @@
      *
      * @return The strategy to be applied when {@code null} is passed as source value to mapping methods.
      */
-    NullValueMappingStrategy nullValueMappingStrategy() default NullValueMappingStrategy.DEFAULT;
+    NullValueMappingStrategy nullValueMappingStrategy() default NullValueMappingStrategy.RETURN_NULL;
+
+    /**
+     * The strategy to use for applying {@code @Mapping} configurations of prototype methods in the interface annotated
+     * with this annotation. If no strategy is configured, {@link MappingInheritanceStrategy#WHEN_EXPLICITLY_REFERENCED}
+     * will be used as default.
+     *
+     * @return The strategy to use for applying {@code @Mapping} configurations of prototype methods in the interface
+     *         annotated with this annotation.
+     */
+    MappingInheritanceStrategy mappingInheritedFromConfig()
```

Comment:
WDYT about naming it `mappingInheritanceStrategy()` to be consistent with the style of the other options?